README.md

```diff
@@ -1,55 +1,49 @@
-# Helpful maintenance scripts
+# Drupal maintenance scripts
 
-A group of helpful scripts for automating common maintenance tasks.
+A group of scripts for automating common maintenance tasks for drupal repos.
 Inspired by [this blogpost.](https://blog.danslimmon.com/2019/07/15/do-nothing-scripting-the-key-to-gradual-automation/)
 
-The initial focus was on updating core or specific contrib modules for all the
-repos, before periodic updates were automated in github actions.
+There are four separate scripts:
 
-It's now more useful for following the steps of a deployment process, modifying
-a few sites at once, and running audits.
+* `common_changes.sh` For making the same change to more than one repo.
 
-## Requirements
-Most requirements - docker, composer, git, curl should already be in place.
-The module audit script requires `libxml2-utils`:
-on Ubuntu `sudo apt install libxml2-utils`
+* `deployment_steps.sh` Test, prep and complete deployments.
+** 1. test with vrt - comparison of prod and dev
+** 2. send communications, update Jira tickets
+** 3. merge to main
+** 4. create tags
+** 5. stage deploy
+** 6. send more communications
+** 7. prod deploy - including post-deployment tests
+
```

Comment:
Should this display as a nested list? It's still easy to read as a sentence, but I thought perhaps some formatting was missing.

Reply:
Yes, it should, I hadn't checked it in a markdown reader - thanks for picking that up.